src/commands/dev/mod.rs

```diff
@@ -19,12 +21,19 @@ pub enum DevSubcommand {
         #[arg(default_value = ".")]
         path: PathBuf,
     },
+    /// Edit the config file from the vault
+    ConfigSecretEdit {
```

Comment:
According to the RFC, the command should be `config-secrets-edit`. Hence the enum should be `ConfigSecretsEdit`.

```suggestion
    ConfigSecretsEdit {
```